bazel/BUILD.plasma

```diff
@@ -0,0 +1,82 @@
+load("@com_github_google_flatbuffers//:build_defs.bzl", "flatbuffer_cc_library")
+
```

Comment:
Is the `bazel/BUILD` file necessary?

Reply:
yes, so Bazel can find the plasma build file, which now needs to be a file registered with Bazel (this is needed since the new version of the new_git_command, but is backward compatible fortunately)